vmdb/config/locales/en.yml

```diff
@@ -940,6 +948,8 @@ en:
       ems_folders:                 Folders
       ems_infra:                   Infrastructure Provider
       ems_infras:                  Infrastructure Providers
+      ems_container:               Provider
```

Comment:
@abonas i do see https://github.com/ManageIQ/manageiq/blob/master/vmdb/public/images/icons/new/vendor-kubernetes.png already present in the master repo.